packages/docs/developer-resources/overview.md

```diff
@@ -3,12 +3,45 @@
 This section contains information about some of the key tools and resources that will help developers start building applications
 on Celo.
 
-{% page-ref page="forno/" %}
+## Tools
 
-{% page-ref page="start.md" %}
```

Comment:
noticed this link was removed altogether, was that intentional?

Reply:
ah good catch, thanks. ill add it to the page